guide/start/policies.md

```diff
@@ -9,169 +9,438 @@ children:
-      maxPoolSize: 3
-      
-- serviceType: brooklyn.entity.database.mysql.MySqlNode
-  id: db
-  name: mysql
-  brooklyn.config:
-    creationScriptUrl: https://bit.ly/brooklyn-visitors-creation-script
+Deploy the app:
+
+{% highlight bash %}
+$ br deploy mycluster.yaml
 {% endhighlight %}
 
-Explore this app using the ```application``` and other commands from the previous section.
+<pre>
+ Id:       nGY58ZZN   
+ Name:     Tomcat Cluster   
+ Status:   In progress   
+</pre>
+
+And wait for the app to be running, viewing its state with:
 
-## Configuring Dependencies
-The App above illustrates how one component in a blueprint can be configured with information relating to one of the other 
-components in the blueprint.  In this example the web cluster is configured with a URL for JDBC connections to the database.
-{% highlight yaml %}
-java.sysprops: 
-      brooklyn.example.db.url: $brooklyn:formatString("jdbc:%s%s?user=%s\\&password=%s",
-         component("db").attributeWhenReady("datastore.url"), "visitors", "brooklyn", "br00k11n")
+{% highlight bash %}
+$ br application
 {% endhighlight %}
 
-the syntax ```$brooklyn:formatString(...)``` is an example of the Brooklyn DSL (Domain Specific Language) which 
-allows expressions referring to Brooklyn's management information to be embedded in blueprints.  The line above also illustrates the use of Brooklyn's ```component(...)``` and ```attributeWhenReady(...)``` to get an identified component from a deployment, and to wait until the component is fully deployed before reading one of its sensors ("datastore.url" in this case). 
+<pre>
+ Id         Name             Status    Location   
+ nGY58ZZN   Tomcat Cluster   RUNNING   Mf0CJac6   
+</pre>
+
+You can view the list of entities within the cluster with the command below (which drills into the 
+application named "Tomcat Cluster", then into its child entity named "Cluster", and then lists its
+entities):
+
+{% highlight bash %}
+$ br application "Tomcat Cluster" entity "Cluster" entity
+{% endhighlight %}
+ 
+<pre>
+ Id         Name            Type   
+ dYfUvLIw   quarantine      org.apache.brooklyn.entity.group.QuarantineGroup   
+ tOpMeYYr   Tomcat Server   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+</pre>
+
+The "quarantine" entity is used when Tomcat servers fail to start correctly - this entity is by 
+default added to the quarantine group, where it can later be investigated. This can be disabled using
+the configuration `dynamiccluster.quarantineFailedEntities: false`.
+
+
+## Tomcat auto-repair
 
-## Managing with Policies
+Each Tomcat entity has a `ServiceFailureDetector` enricher and a `ServiceRestarter` policy. 
 
+An *enricher* generates new events or sensor values (i.e. metrics), e.g. by aggregating existing 
+metrics; it enriches the sensor data available for that entity. A *policy* performs 
+runtime management: it performs actions on an entity (or entities), for example to keep the system 
+in a healthy state.
```

Comment:
How about

An *enricher* generates new events or sensor values (metrics) which it uses to populate the sensor data available for a specific entity. A *policy* coordinates the runtime management of entities by carrying out actions initiated by specific triggers. This may be to keep the system in a healthy state, or autoscale a cluster for example.